doc/nrf/protocols/matter/end_product/last_fabric_removal_delegate.rst

```diff
@@ -0,0 +1,93 @@
+.. _ug_matter_last_fabric_removal_delegate:
+
+Matter fabric removal
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. matter_last_fabric_removal_description_start
+
+When a Matter device is commissioned by the Matter controller, it then belongs to a specific Matter fabric.
+To ensure the interoperability of multiple applications and ecosystems, the device can join the Matter fabrics created by multiple ecosystems using the multi-fabric feature.
+As a result, the device can belong to multiple fabrics and can be managed by multiple Matter controllers.
+To remove the device from a fabric, a Matter controller sends the leave request to it and then removes all information collected from the device.
```

Comment:
```suggestion

To remove the device from a fabric, a Matter controller sends the leave request to it and then removes all information collected from the device.
```
Adding a paragraph break since joining and leaving a fabric is contextually different enough.